lib/interceptor.js

```diff
@@ -32,7 +32,13 @@ module.exports = class Interceptor {
     const uriIsStr = typeof uri === 'string'
     // Check for leading slash. Uri can be either a string or a regexp, but
     // we only need to check strings.
-    if (uriIsStr && /^[^/*]/.test(uri)) {
+
+    if (
+      uriIsStr &&
+      !uri.startsWith('/') &&
+      !uri.startsWith('*') &&
+      !scope.scopeOptions.filteringScope
```

Comment:
Thanks! This is a lot clearer.

In terms of the logic, I'd suggest moving `!scope.scopeOptions.filteringScope` to be the first condition, and adding a comment explaining that it's being checked because `filteringScope` ignores the passed-in URL entirely.